docs/traceability/index.md

```diff
@@ -262,7 +262,7 @@ apic_traceability_agent:
 output.traceability:
   enabled: true
   hosts:
-   - ${TRACEABILITY_HOST:"ingestion-lumberjack.datasearch.axway.com:453"}
+   - ${TRACEABILITY_HOST:"ingestion.datasearch.axway.com:5044"}
```

Comment:
is this changing for EU as well?

Reply:
Yes. 
ingestion.visibility.eu-fr.axway.com:5044

Reply:
Then we need to update the cli for this as well

Reply:
Do we need this url in the cli as well?

Reply:
I created a sub-task for CLI changes. I will address those Monday and touch base w you.